experiments/max_tool_per_agent/multi-builtintools.py

```diff
@@ -0,0 +1,75 @@
+from llama_stack_client.lib.agents.agent import Agent
+from llama_stack_client.lib.agents.event_logger import EventLogger
+from llama_stack_client.types.agent_create_params import AgentConfig
+from termcolor import cprint
+import os
+from llama_stack_client import LlamaStackClient
+from dotenv import load_dotenv
+
+# Load .env file
+load_dotenv()
+
+# this script is tested for llama-stack version 0.1.6
+"""
+How to run:
+step 1. ollama run llama3.2:3b-instruct-fp16 --keepalive 60m
+step 2. export INFERENCE_MODEL="meta-llama/Llama-3.2-3B-Instruct"
+step 3. export LLAMA_STACK_PORT=8321
+step 4. llama stack run --image-type conda ~/llama-stack/llama_stack/templates/ollama/run.yaml
+    (I'm using conda env, follow this if not using conda, https://llama-stack.readthedocs.io/en/latest/distributions/building_distro.html)
+step 5. run the example script
+"""
+
+
+# Access the environment variables
+inference_model = os.getenv("INFERENCE_MODEL")
+llama_stack_port = os.getenv("LLAMA_STACK_PORT")
+ollama_url = os.getenv("OLLAMA_URL")
+tavily_search_api_key = os.environ["TAVILY_SEARCH_API_KEY"]
+wolfram_api_key=os.environ["WOLFRAM_ALPHA_API_KEY"]
```

Comment:
I think it would be a good idea to keep the way you are accessing your environment variables consistent here. Either use `os.getenv()` or `os.environ[]` for all the variables here. I think they both do the same thing.